content/news/2024-05-27-gtn-downloader/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'GTN-downloader'
+date: '2024-05-27'
+tease: "Automating Galaxy Training Network (GTN) data downloads and providing effortless access to GTN data in a Galaxy instance (powered by the OneData cloud)."
+hide_tease: false
+authors: Polina Polunina
+subsites: [global, all]
+main_subsite: eu
+---
+
+# GTN Downloader: Simplifying Access to Galaxy Training Network Data
+
+[GTN-Downloader](https://github.com/usegalaxy-eu/gtn-downloader) makes it easier for users to access and organize data from the Galaxy Training Network (GTN). Here’s a straightforward overview of what this feature does and how it can benefit you.
+
+## What is the GTN Downloader?
+
+The GTN Downloader is a Python script that automates the download of data from GTN tutorials. It goes through the tutorials in the GTN repository, finds `data-library.yaml` files, and creates a structured directory based on the tutorial names and file contents.
+
+### Key Features:
+- **Automated Data Download**: The script finds `data-library.yaml` files in the GTN repository and downloads the associated data files.
+- **Structured Organization**: It creates directories based on the tutorial names and the information in the `data-library.yaml` files, so the files are organized.
+- **Download Summary**: It generates a `download-summary.tsv` file, which includes metadata about the downloaded files, a download report (error, success, already downloaded), and the overall size of the files.
+
+## Seamless Integration with OneData
+
+In addition to local downloads, the GTN Downloader can upload data to OneData, a cloud storage solution. This integration ensures that the latest GTN data is always available to users.
+
+### Automated Workflow with GitHub CI/CD:
+- **Automated Workflow**: A GitHub Actions workflow runs once a week on weekends to download the latest data from the GTN tutorials and upload it to OneData.
+- **Environment Setup**: The workflow sets up necessary environment variables and installs dependencies, including Oneclient, the OneData client.
+- **Data Upload**: After downloading the data, the workflow uploads it to OneData, making it accessible via the cloud.
+
+## Accessing GTN Data in the Cloud
```

Comment:
I would move this section up, I guess this is what we want to advertise the most.